features/step_definitions/book_index_page_steps.rb

```diff
@@ -0,0 +1,30 @@
+Given(/^there are (\d+) books in the database$/) do |number|
```

Comment:
Unused block argument - `number`. You can omit the argument if you don't care about it.